R/minimize.R

```diff
@@ -152,54 +152,495 @@ setMethod("print", signature("adoptrOptimizationResult"),
 #'
 #' The optimization method \code{\link{minimize}} requires an initial
 #' design for optimization.
-#' The function \code{get_initial_design} provides an initial guess based on a
-#' fixed design that fulfills constraints on type I error rate and power.
-#' Note that a situation-specific initial design may be much more efficient.
+#' This function provides a variety of possibilities to hand-craft designs that 
+#' fulfill type I error and type II error constraints which may be used as initial designs.
 #'
 #' @param theta the alternative effect size in the normal case, the
 #' rate difference under the alternative in the binomial case
 #' @param alpha maximal type I error rate
-#' @param beta maximale type II error rate
-#' @param type is a two-stage, group-sequential, or one-stage design requried?
+#' @param beta maximal type II error rate
+#' @param type_design type of design
+#' @param type_c2 either linear-decreasing c2-function according to inverse normal 
+#' combination test or constant c2
+#' @param type_n2 design of n2-function
 #' @param dist distribution of the test statistic
+#' @param cf first-stage futility boundary
+#' @param ce first-stage efficacy boundary. Note that specifying this boundary 
+#' implies that the type I error constraint might not be fulfilled anymore
+#' @param info_ratio the ratio between first and second stage sample size
+#' @param slope slope of n2 function
+#' @param weight weight of first stage test statistics in inverse normal combination test
 #' @param order desired integration order
 #' @template dotdotdot
 #'
 #' @details
 #' The distribution of the test statistic is specified by \code{dist}.
 #' The default assumes a two-armed z-test.
-#'
-#' @return An object of class \code{\link{TwoStageDesign}}.
```

Comment:
As far as I know, CRAN requires each function to have a return value. Please provide it here.

Reply:
Good spot! Fixed it now

Reply:
Thanks!